src/networks.ts

```diff
@@ -59,8 +59,11 @@ interface PublicNetwork {
   nativeToken: string;
   publicRPC: string; // RPC provider of last resort.
   blockExplorer: string;
+  cctpDomain: number;
 }
 
+const CCTP_NO_DOMAIN = -1;
```

Comment:
I think you'll either want to export this value or export a function `isCctpEnabledForNetwork`